packages/nimble-components/src/spinner/specs/README.md

```diff
@@ -0,0 +1,138 @@
+# Nimble Spinner
+
+## Overview
+
+Spinner component (`nimble-spinner`): Indeterminate progress indicator / loading indicator. Meant to be temporarily used in a region of a web page until that region's contents have loaded, or to represent an ongoing operation that doesn't have a known/ determinate duration.
+
+### Background
+
+[Nimble spinner: #346](https://github.com/ni/nimble/issues/346)  
+[Nimble spinner design: #822](https://github.com/ni/nimble/issues/822)  
+[Visual Design spec - Adobe XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/dece308f-79e7-48ec-ab41-011f3376b49b/)
+
+### Non-goals
+
+Ability to show determinate progress:
+
+-   The current visual design for the spinner is distinct/different from an indeterminate radial progress bar
+-   We don't currently have visual designs for a linear or radial progress bar with determinate progress
+-   If/when we want progress bar components in the future, at that point we'll probably create new Nimble components which extend the corresponding FAST components:
+    -   `nimble-progress` or `nimble-linear-progress`, deriving from `fast-progress`, for a linear progress bar; and
+    -   `nimble-progress-ring` or `nimble-radial-progress`, deriving from `fast-progress-ring`, for a radial progress bar
+
+### Features
+
+-   Show indeterminate progress with an animation
+    -   Cannot be paused - assumption is that component will be added to/ removed from the page as needed
+-   Supports 3 sizes: `small` (16x16), `medium` (32x32) (default), `large` (64x64)
+-   For the color theme only, supports an alternate appearance/color scheme via `appearance-variant="alternate"`
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+SystemLink Enterprise uses a spinner based on the Material `mat-spinner`:  
+![SystemLink Material spinner](SlMatSpinner.gif)
+
+A variant of this spinner design is already implemented in the Web Config tool written in Blazor:  
+![Blazor Web Config spinner](BlazorWebConfigSpinner.gif)
+
+## Design
+
+The `nimble-spinner` is an animating indicator that can be placed in a particular region of a page to represent loading progress, or an ongoing operation, of an indeterminate / unknown duration.
+
+### API
+
+Component Name: `nimble-spinner`
+
+Properties/Attributes:
+
+-   `size`
+    -   String attribute controlling size of the component
+    -   Backed by an enum `SpinnerSize` with values `small` (16x16), `medium` (32x32), `large` (64x64)
+    -   If omitted, the default is `medium` (32x32)
+-   `appearance-variant`
+    -   default (undefined): Standard appearance (light/dark/color themes)
+    -   `alternate`: Alternate appearance/color, for the color theme only
+    -   Backed by enum `SpinnerAppearanceVariant`
```

Comment:
Good point, `theme-variant` seems reasonable. I thought about `prominent` but felt like it was being overloaded a bit, but I guess it actually makes sense because the alt one is the only non-gray color (and that's similar to how it's used with breadcrumbs anyway).

Reply:
Updated the spec: switched to `theme-variant="prominent"` for the alt color theme appearance that uses Power Green. Marking this as resolved and removed from Open Issues, if anyone's opposed to the updated plan they can comment.